<launch>
    <arg name="veh" doc="Name of vehicle. ex: megaman"/>
    <arg name="map_name" default="tiles_226" />
	<param name="map_name" value="$(arg map_name)" />
    <param name="veh" value="$(arg veh)"/>

	<node
        ns="$(arg veh)"
		name="graph_search_server"
		pkg="navigation"
		type="graph_search_server.py"
		output="screen"
		required="true"
	/>

    <node
        ns="$(arg veh)"
		name="actions_dispatcher"
		pkg="navigation"
		type="actions_dispatcher.py"
		output="screen"
		required="true"
	/>

</launch>
